00-HTML-CSS-basics/iv.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta name="viewport" content="name=value,name=value" />
+    <meta charset="utf-8">
+    <title>Table</title>
+    <style type="text/css">
+      
+    th {
```

Comment:
Please, do not use HTML tags. I know this is before CSS but anyway exercise didn't asked for it, is that true? So if you add something extra of what the exercise asks for you should do it the best way you can and you know how to create external stylesheets.